packages/core/src/services/documentLogs/_createDocumentLogQuery.ts

```diff
@@ -19,7 +22,7 @@ export function createDocumentLogQuery(workspaceId: number, db = database) {
         .as('cost_in_millicents'),
     })
     .from(scope)
-    .innerJoin(providerLogs, eq(providerLogs.documentLogUuid, scope.uuid))
+    .leftJoin(providerLogs, eq(providerLogs.documentLogUuid, scope.uuid))
```

Comment:
this is matching now all records in scope, is that what you want? we are increasing the size of this query exponentially

Reply:
Other wise I don't catch logs without providerLogs. This can happen now that we capture errors before hitting the AI

Reply:
buf, we need to fix pagination of document logs then, the list of document logs needs to be filtered at this point otherwise any query using this cte will be crazy slow with big numbers